src/libraries/tests.proj

```diff
@@ -7,6 +7,10 @@
     <CoverageReportDir>$(ArtifactsDir)coverage</CoverageReportDir>
     <SerializeProjects>true</SerializeProjects>
   </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworkIdentifier>.NETCoreApp</TargetFrameworkIdentifier> 
```

Comment:
Does it make more sense to just set TargetFramework?  I suspect that is how this was working before.